### PR TITLE
Resize application to be more mobile friendly

### PR DESCRIPTION
**Pull request by oxyflush**
Uses containers from reactstrap to place the application in the center of the screen. These containers are resizable with window size and help the application resize its elements as the window shrinks.

Additionally, cards had a manual width set before which doesn't play well with the container elements. These have been fixed to also scale down by setting the max width to the previous width and letting it scale down in accordance to screen size.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,11 +3,13 @@ import { Provider } from "react-redux";
 import store from "./redux/store";
 import { Header } from './components/Common';
 import { routes } from "./routes"
+import { Container, Row, Col } from 'reactstrap'
 
 function App() {
   return (
     <div>
       <Header />
+      <Container><Row><Col></Col><Col xs="auto">
```
Comment (NickPriv) [inline src/App.js:12]: Would be more readable on separate lines

**Review by NickPriv — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (oxyflush) [reply src/App.js]: Done in https://github.com/NickPriv/FryRankFrontend/pull/17/commits/76558e74dd8d9ef5e5e159ffc1d5924c14bfe231.

**Review by NickPriv — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 2]
```diff
@@ -3,27 +3,36 @@ import { Provider } from "react-redux";
...
+                                  key={key}
+                                  component={route.component}
+                              />
+                            ))}
+                    </Switch>
+                  </Router>
+                </Provider>
+              </div>
+          </Col>
+          <Col></Col>
```
[the remark this answers is not in the document]
Comment (oxyflush) [reply src/App.js]: https://github.com/NickPriv/FryRankFrontend/pull/17/commits/d736fb173ac8f711e8d178dfd89f20444a88816a

File: src/App.js [hunk 3]
```diff
@@ -3,27 +3,36 @@ import { Provider } from "react-redux";
...
-                          component={route.component}
-                      />
-                    ))}
-            </Switch>
-          </Router>
-        </Provider>
-      </div>
+      <Container>
+      <Row>
+          <Col></Col>
```
[the remark this answers is not in the document]
Comment (oxyflush) [reply src/App.js]: https://github.com/NickPriv/FryRankFrontend/pull/17/commits/d736fb173ac8f711e8d178dfd89f20444a88816a

File: src/App.js [hunk 4]
```diff
@@ -3,27 +3,36 @@ import { Provider } from "react-redux";
...
+                    <Switch>
+                      {routes.map((route, key) => (
+                              <Route
+                                  exact
+                                  path={route.path}
+                                  key={key}
+                                  component={route.component}
+                              />
+                            ))}
+                    </Switch>
```
[the remark this answers is not in the document]
Comment (oxyflush) [reply src/App.js]: https://github.com/NickPriv/FryRankFrontend/pull/17/commits/d736fb173ac8f711e8d178dfd89f20444a88816a

**Review by NickPriv — APPROVED** [review]

**Pull request merged**